.github/workflows/ci.yaml

```diff
@@ -172,20 +173,6 @@ jobs:
             os_version: 15.4
           - toxenv: metadata
             os_version: 15.4
-          - toxenv: all
```

Comment:
Accidental removal?

Reply:
no it's not accidental, the `all` environment should be running by default already

Reply:
nit: it would be nice to split into another commit with proper justification.